### PR TITLE
✨ New navigation featuring Pipelines and TA side-by-side

### PR DESCRIPTION
**Pull request by olyism**
New navigation UI that shows Pipelines and Test Analytics side-by-side. This PR replaces the existing sidebar navigation paradigm.

See the 🎦 Loom video https://www.loom.com/share/0f94be9f72804b5eb37e5f4b1a3f342f or read summary below:

Summary of changes:

- [x] Manage navigation content in YAML
- [x] New header and navigation UI
- [x] Pills in sidebar nav
- [x] Remove old sidebar code
- [x] Scroll padding for anchor links
- [x] Add notification message advising of recent big changes
  - [ ] Change wording in notification message (see Linear ticket)

## Excluded from this PR

To keep this PR as small as possible, the following features will come later:

- Breadcrumbs
- New header on homepage
- Not changing any pages' paths relative to the new structure

## Screenshots

Screen Shot 2022-06-15 at 1 20 54 pm

Screen Shot 2022-06-15 at 1 21 16 pm

### CODE REVIEW

**Comment by olyism** [comment]
Not sure why I'm getting a 500 error on the preview. I hope this won't cause a problem on the production site 😬. 

Works on my local dev. Any ideas? 🤷‍♂️

**Comment by olyism** [comment]
Pulled back to draft to add notification message advising customers of major navigation changes.

**Comment by olyism** [comment]
Added notification message in commit https://github.com/buildkite/docs/pull/1591/commits/a0efe723c8610e0c99ac1888a2453598ecae96aa. Ready for review again.

- Add notification content in `data/notification.yml`
- Keep notification dismissed state in localStorage
- Four different style options (i.e. success, info, warning, and danger)

Screen Shot 2022-06-16 at 1 50 05 am

**Review by karensawrey — COMMENTED** [review]
Whoa, this is many changes in a single PR :) 
For me, clicking on any link on the homepage of the preview results in the following: 
under-maintenance

Another thing I spotted in the Loom - the product buttons at the top say "Pipeline" and "Test Analytics" -- "Pipeline" should be changed to "Pipelines". 

It would be truly amazing if you could add a couple of sentences on notification & warning types to the style guide, too. Thanks!

**Comment by olyism** [comment]
Thanks @karensawrey I'll take a look at the typo fixes.

The 500 server error is really strange 🤔 I thought pushing a commit again would fix it. It works on my local dev, but will investigate to double make sure. 🔍

**Comment by olyism** [comment]
500 error resolved in https://github.com/buildkite/docs/pull/1591/commits/8c4cad3b0541187a575abe909ddc76d61ce0a8be

File: data/notification.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: |
+  We've recently updated the navigation
+desc: |
+  You can find Pipelines, Test Analytics, and anything else across the top.
```
Comment (karensawrey) [inline data/notification.yml:5]:
```suggestion
  You can find Pipelines and Test Analytics in their own sections in the new horizontal menu bar at the top of the page.
```

Comment (karensawrey) [reply data/notification.yml:5]: Suggested by @plaindocs & that's exactly the fix I wanted here.

**Review by karensawrey — APPROVED** [review]

**Review by lachlanhardy — APPROVED** [review]
@olyism This PR is a work of art, truly. Thanks so much for this phenomenal attention to detail. So easy to review commit by commit.

Ship it, I say!

**Pull request merged**